templates/helpers/dsl-imports.hbs

```diff
@@ -0,0 +1,25 @@
+{{#ai-context}}
+    /**
+    * STORM-AI: Below you'll find all the available entities already imported. Remove the imports that are not needed in your implementation.
```

Comment:
I wonder if it would make sense to remove this comment and just keep them until the code fixing phase is done 🤔

Reply:
We can experiment